contrib/google.golang.org/api/api_test.go

```diff
@@ -6,15 +6,15 @@
 package api
 
 import (
-	"io/ioutil"
+	"io"
 	"net/http"
 	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
-	books "google.golang.org/api/books/v1"
-	civicinfo "google.golang.org/api/civicinfo/v2"
-	urlshortener "google.golang.org/api/urlshortener/v1"
```

Comment:
This is unrelated to your change. Can you please revert this part of the PR?

Reply:
Ok. Let me revert those changes and stick to specifically the ioutil changes in this PR 🙂